src/spotbugs/excludesFilter.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+  <!--
+    Exclusions in this section have been triaged and determined to be
+    false positives.
+  -->
+  <Match>
```

Comment:
Can an line `@SuppressFBWarnings` on the method or class be used instead directly in the source file rather than having a separate XML file? I prefer to keep all things in one place.

Reply:
Yes, individual suppressions can be used, though that will scatter many more suppressions in the source files.  In particular, the `NM_METHOD_NAMING_CONVENTION` suppression will need to find the right location to insert that suppression for variable names without suppressing all checks for inconsistently named variables.

I've had a mixed experience with inline suppression of spotbugs warnings.  When I want to provide a detailed reason for a method or field level suppression, inline suppressions have worked reasonably well.  When I wanted a more general suppression, the XML configuration file seemed to work better as a central location to configure suppressions.  Both techniques work and each has their own strengths and weaknesses.

I'm not sure that I'll have time to move from centrally defined suppressions to suppressions inside the individual files for this repository.  This pull request was a side effect of #598 and already took more time than I had planned.  I'm not sure that I have additional time to spend on this side effect.  If you'd like to make that change, you're welcome to do it.  You're welcome to use any or all of this pull request as a basis for the change.  Thanks for maintaining the Jira plugin!